plugins/apitracing/src/lib/FunctionHook.cpp

```diff
@@ -56,24 +57,62 @@ namespace ApiTracing
             }
 
             auto extractedParameters = extractor->extractParameters(event, parameterInformation);
-            logParameterList(extractedParameters);
+
+            Json::Value root;
+            Json::StreamWriterBuilder builder;
+            builder["indentation"] = "";
+            root["functionName"] = functionName;
+            auto json = getParameterListAsJson(extractedParameters, root);
+            std::string unformattedTraces = Json::writeString(builder, json);
+
+            logger->info("Monitored function parameters",
+                         {{"ProcessDtb", fmt::format("{:x}", event.getCr3())},
+                          {"ThreadID", event.getGs()},
```

Comment:
This is confusing, because thread IDs exist in Windows, but there are not stored directly in the GS register. The GS register stores the base address of the TEB instead.